dashboard/app/controllers/levels_controller.rb

```diff
@@ -211,6 +211,8 @@ def new
         @game = Game.eval
       elsif @type_class <= Applab
         @game = Game.applab
+      elsif @type_class <= GamelabJr
+        @game = Game.spritelab
```

Comment:
Are you sure this is needed? It may not be, since we unconditionally set game type in `create!(...)` in gamelab_jr.rb below.

Why there are two different ways of setting the default, I don't know... 🤷‍♂️